src/clang-c-frontend/clang_c_convert.h

```diff
@@ -85,13 +87,26 @@ class clang_c_convertert
   bool convert_builtin_types();
   bool convert_top_level_decl();
 
+  // Since this class is inherited by clang-cpp-frontend,
+  // some get_* functions are made `virtual' to deal with clang CXX declarations
   virtual bool get_decl(const clang::Decl &decl, exprt &new_expr);
 
   virtual bool get_var(const clang::VarDecl &vd, exprt &new_expr);
 
   virtual bool get_function(const clang::FunctionDecl &fd, exprt &new_expr);
 
-  bool get_function_params(const clang::ParmVarDecl &pd, exprt &param);
+  virtual bool
+  get_function_body(const clang::FunctionDecl &fd, exprt &new_expr);
+
+  // Parse function parameters
```

Comment:
Can I ask you to follow the standard for comments as in: https://github.com/esbmc/esbmc/blob/master/src/goto-symex/goto_symex_state.h?